src/Options/ContactDetails.php

```diff
@@ -0,0 +1,137 @@
+<?php namespace Leean\Elements\Options;
+
+/**
+ * Class ContactDetails.
+ *
+ * @package Leean\Elements\Options
+ */
+class ContactDetails
```

Comment:
We already have an api for menus - they're in the static endpoint.
However, I' not sure this is the right approach for social. How do we tie the menu items to an icon in the front end for example?